apps/extension/src/ui/apps/onboard/routes/AddAccount/AccountAddDerivedPage.tsx

```diff
@@ -1,10 +1,15 @@
 import { AccountAddDerivedForm } from "@ui/domains/Account/AccountAdd/AccountAddDerived/AccountAddDerivedForm"
+import { useTranslation } from "react-i18next"
 
 import { AccountAddWrapper } from "./AccountAddWrapper"
 
-export const AccountAddDerivedPage = () => (
-  <AccountAddWrapper
-    title="Create a new account"
-    render={(onSuccess) => <AccountAddDerivedForm onSuccess={onSuccess} />}
-  />
-)
+export const AccountAddDerivedPage = () => {
+  const { t } = useTranslation("admin")
+
+  return (
+    <AccountAddWrapper
+      title={t("Create a new account")}
```

Comment:
The `t` function is called on the props given inside `AccountAddWrapper`. I wasn't entirely sure if this would work though and hadn't tested it yet. If it doesn't work, we should also update `AccountAddWrapper` to remove the `t` call there.